src/query/expression/src/converts/arrow/to.rs

```diff
@@ -187,6 +188,13 @@ impl From<&TableField> for Field {
                 );
                 ArrowDataType::LargeBinary
             }
+            TableDataType::Interval => {
+                metadata.insert(
+                    EXTENSION_KEY.to_string(),
+                    ARROW_EXT_TYPE_INTERVAL.to_string(),
+                );
+                ArrowDataType::LargeBinary
```

Comment:
We can use `FixedSizeBinary` instead of `LargeBinary`, because the interval value has a fixed length, which will remove the offset and reduce the column size.